Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccreissuance/core/server/data/DccReissuanceErrorResponse.kt

```diff
@@ -0,0 +1,8 @@
+package de.rki.coronawarnapp.dccreissuance.core.server.data
+
+import com.google.gson.annotations.SerializedName
+
+data class DccReissuanceErrorResponse(
+    @SerializedName("error") val error: String,
```

Comment:
`errorCode` as per [specs]( https://github.com/corona-warn-app/cwa-app-tech-spec/pull/154/commits/e0302a180bfa9aad8693f3835171d711f3318898#diff-b503751806bd910ffe10c62a514f15f69ed3be758d95ecff6e8700e1cab5b5cbR183)

Reply:
small unit test for parsing the error would be appreciated

Reply:
Please read the description.
There is a missmatch between the errorCodes & and the Codes in the Spec. Max will provide an updated one and then the spec will match to this PR.

Reply:
Ok 👍